js/menus/menuTemplate.js

```diff
@@ -156,9 +171,73 @@ function getTemplate(app) {
                 role: 'front'
             }
         ]
+    } else {
+      // Window menu when Windows.
+        template[2].submenu.push(
+            {
+                label: 'Auto Launch on Windows Startup', 
+                type: 'checkbox', 
+                checked: launchOnStartup,
+                click: function (item) {
+                    if (item.checked){
+
+                        symphonyAutoLauncher.isEnabled()
```

Comment:
i need to confirm with product but believe expectation here is that auto launch is enabled after installing the app.  we might have to invoke the app during install process with special flag that sets launch values then exits immediately.